instance-scheduler/environments.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"encoding/json"
+	//"errors"
+	//"log"
+	"strings"
+	"fmt"
+    "net/http"
+    "net/url"
+    "io/ioutil"
```

Comment:
io/ioutil has been [deprecated](https://pkg.go.dev/io/ioutil), can swap this for the io package ♻️ 

```suggestion
    "io"
```

Reply:
Nice spot.